types/Link.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+var GraphQL = require('graphql');
+
+var KeystoneLinkType = new GraphQL.GraphQLObjectType({
+	name: 'Keystone Link',
+	fields: {
+		raw: {
+			type: GraphQL.GraphQLString,
+			description: 'The raw unformatted URL',
+		},
+		format: {
```

Comment:
I wouldn't call this `format` - maybe `formatted`, but I don't think we need to make two fields available.

Adding nested fields complicates the GraphQL shape, so we want to avoid it where possible.

In this case, as per the Date field, we would just use an optional `format` argument (in this case a boolean, since there is no config possible for how a link is formatted) that would change the value returned in the top level field.

In any case, the value of formatting URLs in Keystone is pretty ill defined, and I'm happy for now to just always return the stored value of the field. (??)